pkgpanda/cli.py

```diff
@@ -210,16 +210,16 @@ def main():
             # Run all checks
             sys.exit(run_checks(checks, install, repository))
     except ValidationError as ex:
```

Comment:
This breakout into 3 possible informational messages is not really useful in practice IMHO